src/routes/AuthRoutes.jsx

```diff
@@ -1,17 +1,8 @@
-import React from 'react';
-import { Route, Routes } from 'react-router-dom';
-import SignIn from '../components/auth/SignIn';
-import Register from '../components/auth/Register';
-import ForgotPassword from '../components/auth/ForgotPassword';
+import { SignIn, Register, ForgotPassword } from '../components/auth';
 
-const AuthRoutes = () => {
-    return (
-        <Routes>
-            <Route path="/signin" element={<SignIn />} />
-            <Route path="/register" element={<Register />} />
-            <Route path="/passwordreset" element={<ForgotPassword />} />
-        </Routes>
-    );
-};
+export const authRoutes = [
+    { path: "/signin", name: "Sign in", element: <SignIn />, isMenu: false, isAuthenticated: false },
```

Comment:
This is the simplified approach to declaring our routes.
The reason I wanted it done this way is for the ability to define custom properties on the declaration of the route. In this case, I added 'isMenu' and 'isAuthenticated' as additional properties.

These properties can then be used wherever this array gets used